tests/integration/test_ping.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pytest
+
+from tests.helpers import IntegrationTestCaseBase
+
+
+@pytest.mark.integration
+class TestPing(IntegrationTestCaseBase):
+    def setup_method(self):
+        super(TestPing, self).setup_method()
+        self.rebirthdb_host=os.getenv('REBIRTHDB_HOST')
+
+    def teardown_method(self):
+        with self.r.connect(host=self.rebirthdb_host) as conn:
+            curr = self.r.db("rethinkdb").table("users").filter(
+                self.r.row["id"].ne("admin")
+            ).delete().run(conn)
+            assert list(curr)
```

Comment:
Why do we need the assertion here?

Reply:
I dislike having unused variables. It can be removed now that I know tear down is working.

Reply:
I mean the variable assignment can be dropped ^-^